dist/api_data.py

```diff
@@ -413,9 +413,10 @@ def __cmp__(self, other):
         blocking. This option is intended for use with internal stress
         testing of WiredTiger. Only available if WiredTiger is configured
         with --enable-diagnostic. Options are given as a list, such as
-        <code>"diagnostic_timing_stress=[checkpoint_slow]"</code>''',
+        <code>"diagnostic_timing_stress=[checkpoint_slow,
+                                        page_split_slow]"</code>''',
```

Comment:
I'd prefer if this were called `page_split_race` (and that filters through the change)